### ISSUE TITLE
Remove GIBill school feedback layout table, the page doesn't scroll horizontally on mobile and the content is cutoff

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#129353]
### Point of contact

Martha Wilkes

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

When you view this page on an iPhone 
https://www.va.gov/education/submit-school-feedback/introduction

The table of complaint types is not fully visible because the page doesn't scroll horizontally.

<img width="1125" height="2436" alt="Image" src="https://github.com/user-attachments/assets/915a8010-55b6-424f-936a-7241c30947bc" />

### Reproduction steps

View this page on an iPhone 
https://www.va.gov/education/submit-school-feedback/introduction

The table of complaint types is not fully visible because the page doesn't scroll horizontally.

### Proposed solution or next steps

The fix is to simply remove the unnecessary table and just list the complaint types in a bulleted list. 

### References, articles, or WCAG support

Tables should only be used for [displaying tabular data per VA.gov standards, never for layout purposes](https://design.va.gov/components/table#accessibility-considerations):

> - Tables should be used to display tabular data, which is structured data made up of rows and columns. A table allows the information to be easily interpreted by visually associating row and column headers.
> - Do not use tables for layout purposes. Tables should only be used for data that has inherent relationships, not for design purposes.

### Type of issue

- [ ] Axe-core
- [ ] Screenreader | assistive tech & device support
- [ ] Keyboard
- [ ] Voice command
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [x] Components and pattern standards
- [x] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [x] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [x] Something else
